base/utils/R/run.write.configs.R

```diff
@@ -165,8 +166,24 @@ runModule.run.write.configs <- function(settings, overwrite = TRUE) {
     return(PEcAn.settings::papply(settings, runModule.run.write.configs, overwrite = FALSE))
   } else if (PEcAn.settings::is.Settings(settings)) {
     write <- settings$database$bety$write
-    ens.sample.method <- settings$ensemble$method
-    return(run.write.configs(settings, write, ens.sample.method, overwrite = overwrite))
+
+    #Old version of pecan xml which don't have a sampling space if it's not specified either it's an old xml or it's just not especified.
```

Comment:
1. As noted by @infotroph this function has been deprecated (see line 13 above) and changes should be made in PEcAn.workflow.
2. typos
3. the catching and fixing of old pecan.xml tags should be done in the PEcAn.settings module.